Gemfile

```
@@ -4,6 +4,7 @@ source 'https://rubygems.org' # global source
 source 'https://rubygems.pkg.github.com/appfolio' # global source
 
 source 'https://rubygems.org' do
+  gem 'base64', '>=0.2.0'
```

Can we put an upper limit on this range?

```suggestion
  gem 'base64', '>= 0.2', '< 1'
```

---

Done!